tests/acceptance/test_services_list.py

```diff
@@ -0,0 +1,84 @@
+"""
+Contains acceptance tests for the ``/services`` endpoint. This is the first
+endpoint to make use of the DB.
+"""
+import json
+
+from tests.acceptance import AcceptanceTestCaseWithService
+from tests.acceptance.test_cases.acceptance_test_case import AcceptanceTestCase
+
+
+class ServicesEndpointTestCase(AcceptanceTestCaseWithService):
+    """
+    Provides properties common to testing the ``/services`` endpoint
+    """
+    @property
+    def url(self) -> str:
+        """
+
+        :return: The URL to the ``/services`` endpoint
+        """
+        return '%s/services' % self.app_url
+
+    @property
+    def headers(self) -> dict:
+        """
+
+        :return: The headers to use for testing the API
+        """
+        return {'Content-Type': 'application/json'}
+
+
+class TestGetEndpoint(ServicesEndpointTestCase):
+    """
+    Tests that the API is correctly able to retrieve a service from the DB
+    """
+    def test_get(self) -> None:
+        response = self.client.get(self.url, headers=self.headers)
+        self.assertEqual(response.status_code, 200)
+        response_body = json.loads(response.data.decode('utf-8'))
+        self.assert_correct_data(response_body)
+
+    def assert_correct_data(self, response_body: dict) -> None:
+        """
```

Comment:
Give a line here to explain what the correct data is. Could we use a ``Serializer`` in order to get the expected data more quickly?